### PR TITLE
Actually fix meshctl writing a CA file path to in-cluster secrets

### PR DESCRIPTION
**Pull request by GrahamGoudeau**
Previously:

```
➜ ./_output/meshctl --kubeconfig ~/.kube/smh-590-gke-test-config cluster register --remote-kubeconfig ~/.kube/smh-590-gke-test-config --remote-cluster-name management-plane --remote-context gke_service-mesh-hub_us-central1-c_istio-mc-1
Successfully wrote service account to remote cluster...
Starting helm installation
Successful installation!
Successfully set up CSR agent...
Successfully wrote kube config secret to master cluster...

Cluster management-plane is now registered in your Service Mesh Hub installation

service-mesh-hub on  smh-590-real-fix [$?] at ☸️  gke_service-mesh-hub_us-central1-c_istio-mc-1 took 9s
➜ k --kubeconfig ~/.kube/smh-590-gke-test-config -n service-mesh-hub logs deployment/mesh-discovery
{"level":"info","ts":1587405809.3117096,"logger":"mesh-discovery.controller-runtime.metrics","msg":"metrics server is starting to listen","version":"0.4.7","addr":":8080"}
{"level":"info","ts":1587405809.3124855,"logger":"mesh-discovery.controller-runtime.manager","msg":"starting metrics server","version":"0.4.7","path":"/metrics"}
{"level":"info","ts":1587405809.3151639,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting EventSource","version":"0.4.7","controller":"multi-cluster-controller","source":"kind source: /, Kind="}
{"level":"info","ts":1587405809.4186373,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting Controller","version":"0.4.7","controller":"multi-cluster-controller"}
{"level":"info","ts":1587405809.418703,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting workers","version":"0.4.7","controller":"multi-cluster-controller","worker count":1}
{"level":"error","ts":1587405809.421871,"logger":"mesh-discovery","msg":"error adding member cluster for secret management-plane.service-mesh-hub: invalid configuration: unable to read certificate-authority /Users/grahamgoudeau/.kube/smh-590-gke-ca.pem for management-plane due to open /Users/grahamgoudeau/.kube/smh-590-gke-ca.pem: no such file or directory: Failed to convert secret management-plane.service-mesh-hub to rest config: invalid kube config for cluster management-plane in the secret management-plane in namespace service-mesh-hub","version":"0.4.7","stacktrace":"github.com/solo-io/service-mesh-hub/services/common/multicluster/watcher.(*multiClusterHandler).CreateSecret\n\t/workspace/service-mesh-hub/services/common/multicluster/watcher/watcher.go:55\ngithub.com/solo-io/service-mesh-hub/pkg/api/kubernetes/core/v1/controller.genericSecretHandler.Create\n\t/workspace/service-mesh-hub/pkg/api/kubernetes/core/v1/controller/event_handlers.go:91\ngithub.com/solo-io/skv2/pkg/events.(*eventWatcher).Reconcile\n\t/go/pkg/mod/github.com/solo-io/skv2@v0.0.1/pkg/events/watcher.go:92\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.5.1/pkg/internal/controller/controller.go:256\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.5.1/pkg/internal/controller/controller.go:232\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).worker\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.5.1/pkg/internal/controller/controller.go:211\nk8s.io/apimachinery/pkg/util/wait.JitterUntil.func1\n\t/go/pkg/mod/k8s.io/apimachinery@v0.17.2/pkg/util/wait/wait.go:152\nk8s.io/apimachinery/pkg/util/wait.JitterUntil\n\t/go/pkg/mod/k8s.io/apimachinery@v0.17.2/pkg/util/wait/wait.go:153\nk8s.io/apimachinery/pkg/util/wait.Until\n\t/go/pkg/mod/k8s.io/apimachinery@v0.17.2/pkg/util/wait/wait.go:88"}
```

After this change:
```
➜ ./_output/meshctl --kubeconfig ~/.kube/smh-590-gke-test-config cluster register --remote-kubeconfig ~/.kube/smh-590-gke-test-config --remote-cluster-name management-plane --remote-context gke_service-mesh-hub_us-central1-c_istio-mc-1
Successfully wrote service account to remote cluster...
Starting helm installation
Successful installation!
Successfully set up CSR agent...
Successfully wrote kube config secret to master cluster...

Cluster management-plane is now registered in your Service Mesh Hub installation

service-mesh-hub on  smh-590-real-fix [$?] at ☸️  gke_service-mesh-hub_us-central1-c_istio-mc-1 took 8s
➜ k --kubeconfig ~/.kube/smh-590-gke-test-config -n service-mesh-hub logs deployment/mesh-discovery
{"level":"info","ts":1587405902.376711,"logger":"mesh-discovery.controller-runtime.metrics","msg":"metrics server is starting to listen","version":"0.4.7","addr":":8080"}
{"level":"info","ts":1587405902.3771682,"logger":"mesh-discovery.controller-runtime.manager","msg":"starting metrics server","version":"0.4.7","path":"/metrics"}
{"level":"info","ts":1587405902.3778708,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting EventSource","version":"0.4.7","controller":"multi-cluster-controller","source":"kind source: /, Kind="}
{"level":"info","ts":1587405902.4785552,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting Controller","version":"0.4.7","controller":"multi-cluster-controller"}
{"level":"info","ts":1587405902.4786282,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting workers","version":"0.4.7","controller":"multi-cluster-controller","worker count":1}
{"level":"info","ts":1587405921.823688,"logger":"mesh-discovery","msg":"Adding new cluster member: management-plane","version":"0.4.7"}
{"level":"info","ts":1587405921.8237414,"logger":"mesh-discovery","msg":"Number of remote clusters: 1","version":"0.4.7"}
{"level":"info","ts":1587405921.824116,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting EventSource","version":"0.4.7","controller":"management-plane","source":"kind source: /, Kind="}
{"level":"info","ts":1587405921.824266,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting EventSource","version":"0.4.7","controller":"management-plane","source":"kind source: /, Kind="}
{"level":"info","ts":1587405921.8253036,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting EventSource","version":"0.4.7","controller":"mesh-controller","source":"kind source: /, Kind="}
{"level":"info","ts":1587405921.8257043,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting EventSource","version":"0.4.7","controller":"mesh-workload-apps_controller","source":"kind source: /, Kind="}
{"level":"info","ts":1587405921.8243754,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting EventSource","version":"0.4.7","controller":"management-plane","source":"kind source: /, Kind="}
{"level":"info","ts":1587405921.9256802,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting Controller","version":"0.4.7","controller":"management-plane"}
{"level":"info","ts":1587405921.925774,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting workers","version":"0.4.7","controller":"management-plane","worker count":1}
{"level":"info","ts":1587405921.9259903,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting Controller","version":"0.4.7","controller":"mesh-controller"}
{"level":"info","ts":1587405921.9260118,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting workers","version":"0.4.7","controller":"mesh-controller","worker count":1}
{"level":"info","ts":1587405921.9256864,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting Controller","version":"0.4.7","controller":"management-plane"}
{"level":"info","ts":1587405921.9260516,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting workers","version":"0.4.7","controller":"management-plane","worker count":1}
{"level":"info","ts":1587405921.9267888,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting Controller","version":"0.4.7","controller":"mesh-workload-apps_controller"}
{"level":"info","ts":1587405921.9268265,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting workers","version":"0.4.7","controller":"mesh-workload-apps_controller","worker count":1}
{"level":"info","ts":1587405921.9269369,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting Controller","version":"0.4.7","controller":"management-plane"}
{"level":"info","ts":1587405921.9269557,"logger":"mesh-discovery.controller-runtime.controller","msg":"Starting workers","version":"0.4.7","controller":"management-plane","worker count":1}
```

BOT NOTES: 
resolves https://github.com/solo-io/service-mesh-hub/issues/590

### CODE REVIEW

**Comment by GrahamGoudeau** [comment]
@rvennam FYI

File: cli/pkg/common/kube/converter.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+		config.Cluster: rawKubeConfig,
+	}
+
+	return &k8s_core_v1.Secret{
+		ObjectMeta: k8s_meta_v1.ObjectMeta{
+			Labels:    map[string]string{KubeConfigSecretLabel: "true"},
+			Name:      secretName,
+			Namespace: secretNamespace,
+		},
+		Type: k8s_core_v1.SecretTypeOpaque,
```
Comment (yuval-k) [inline cli/pkg/common/kube/converter.go:62]: consider using a custom secret type?

File: cli/pkg/common/kube/converter.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+		Data: secretData,
```
Comment (yuval-k) [inline cli/pkg/common/kube/converter.go:63]: i think Data has to base base64, no?
consider using StringData

File: cli/pkg/common/kube/converter.go (hunk 1 above)
Comment (GrahamGoudeau) [reply cli/pkg/common/kube/converter.go:62]: This isn't new code, just ported over to a struct/interface component from its old implementation. Going to punt on making changes beyond this bugfix for now, but we can definitely open an issue to make this change if we don't have one already

File: cli/pkg/common/kube/converter.go (hunk 2 above)
Comment (GrahamGoudeau) [reply cli/pkg/common/kube/converter.go:63]: This implementation hasn't changed from how it was running previously, just ported over to a new place

File: cli/pkg/common/kube/converter.go (hunk 1 above)
Comment (harveyxia) [reply cli/pkg/common/kube/converter.go:62]: @joekelley is working on porting this over to skv2, so once that stabilizes a bit we can migrate over to skv2 for this functionality

File: cli/pkg/common/kube/converter.go (hunk 2 above)
Comment (harveyxia) [reply cli/pkg/common/kube/converter.go:63]: I'm guessing this works because of `clientcmd.Write(config.Config)`

Comment (yuval-k) [reply cli/pkg/common/kube/converter.go:63]: nvm; data encoding/decoding is probably handled by client

**Pull request merged**